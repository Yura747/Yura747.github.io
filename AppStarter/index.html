<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>AppStarter</title>
	<!-- Fonts -->
	<link href="https://fonts.googleapis.com/css?family=Nunito+Sans:200,300,400,600,700,800,900&display=swap" rel="stylesheet">
	<!-- Favicon -->
	<link rel="apple-touch-icon" sizes="76x76" href="favicon/apple-touch-icon.png">
	<link rel="icon" type="image/png" sizes="32x32" href="favicon/favicon-32x32.png">
	<link rel="icon" type="image/png" sizes="16x16" href="favicon/favicon-16x16.png">
	<link rel="manifest" href="favicon/site.webmanifest">
	<link rel="mask-icon" href="favicon/safari-pinned-tab.svg" color="#5bbad5">
	<meta name="msapplication-TileColor" content="#6570ef">
	<meta name="theme-color" content="#ffffff">
	<!-- my styles -->
	<link rel="stylesheet" href="css/style.css">
</head>
<body>

<section id="section-header-tablet">
	<div class="header-tablet">
		<div class="wrapper">
			<header>
				<div class="header-wr">
					<div class="logo">
						<p>AppStarter</p>
					</div>
					<nav class="desctop-menu">
						<ul>
							<li><a href="#features">Features</a></li>
							<li><a href="#video-tour">Video Tour</a></li>
							<li><a href="#reviews">Reviews</a></li>
							<li><a href="#pricing">Pricing</a></li>
							<li><button class="button white">get it free</button></li>
						</ul>
					</nav>
					<div class="hamburger-wr">
						<div class="hamburger">
							<span class="bar bar1"></span>
							<span class="bar bar2"></span>
							<span class="bar bar3"></span>
							<span class="bar bar4"></span>
						</div>
					</div>
				</div>
				<nav class="mobile-menu-wrapper">
					<div class="mobile-menu">
						<ul>
							<li><a href="#features">Features</a></li>
							<li><a href="#video-tour">Video Tour</a></li>
							<li><a href="#reviews">Reviews</a></li>
							<li><a href="#pricing">Pricing</a></li>
							<li><a href="#get-it-free">get it free</a></li>
						</ul>
					</div>
				</nav>
			</header>
			<div class="columns-wr">
				<div class="column-left">
					<div class="app-icon">
						<img src="icons/AppIcon.png" alt="AppIcon">
					</div>
					<h2 class="title">
						AppStarter. <span>Best landing page for web and mobile apps</span>
					</h2>
					<div class="buttons-wr">
						<button class="button orange">Download Now</button>
						<button class="button white">Watch Video</button>
					</div>
				</div>
				<div class="column-right">
					<img class="img-tablet" src="img/Tablet.png" alt="intro-tablet">
				</div>
			</div>
		</div>
	</div>
</section>
<section id="section-features">
	<div class="features">
		<div class="wrapper">
			<div class="features-wr">
				<h2 class="title">
					How <span>does it work</span>
				</h2>
				<div class="cards-wr">
					<div class="card">
						<div class="card-icon">
							<img src="icons/icon_reload.png" alt="icon-reload">
						</div>
						<div class="card-title">
							<h3 class="title">
								How <span>does it work</span>
							</h3>
						</div>
						<div class="card-content">
							<p>
								This sounded nonsense to Alice, so she said nothing,
								but set off at once toward the Red Queen. 
								To her surprise.
							</p>
						</div>
						<div class="card-link">
							<a href="#">Watch video</a>
						</div>
					</div>
					<div class="card">
						<div class="card-icon">
							<img src="icons/icon_mail.png" alt="icon-mail">
						</div>
						<div class="card-title">
							<h3 class="title">
								All emails <span>in one place</span>
							</h3>
						</div>
						<div class="card-content">
							<p>
								This sounded nonsense to Alice, so she said nothing,
								but set off at once toward the Red Queen. 
								To her surprise.
							</p>
						</div>
						<div class="card-link">
							<a href="#">Watch video</a>
						</div>
					</div>
					<div class="card">
						<div class="card-icon">
							<img src="icons/icon_ring.png" alt="icon-ring">
						</div>
						<div class="card-title">
							<h3 class="title">
								Notifications <span>about sales</span>
							</h3>
						</div>
						<div class="card-content">
							<p>
								This sounded nonsense to Alice, so she said nothing,
								but set off at once toward the Red Queen. 
								To her surprise.
							</p>
						</div>
						<div class="card-link">
							<a href="#">Watch video</a>
						</div>
					</div>
				</div>
			</div>
		</div>
	</div>
</section>
<section id="section-press">
	<div class="press">
		<div class="wrapper">
			<div class="press-wr">
				<div class="sponsor">
					<img src="icons/sponsor_wired.png" alt="wired-logo">
				</div>
				<div class="sponsor">
					<img src="icons/sponsor_lifehacker.png" alt="lifehacker-logo">
				</div>
				<div class="sponsor">
					<img src="icons/sponsor_mashable.png" alt="mashable-logo">
				</div>
				<div class="sponsor">
					<img src="icons/sponsor_theverge.png" alt="the-verge-logo">
				</div>
				<div class="sponsor">
					<img src="icons/sponsor_gizmodo.png" alt="gizmodo-logo">
				</div>
				<div class="sponsor">
					<img src="icons/sponsor_macworld.png" alt="macworld-logo">
				</div>
			</div>
		</div>
	</div>
</section>
<section id="section-highlights_1">
	<div class="highlights_1">
		<div class="wrapper">
			<div class="columns-wr">
				<div class="column-left">
					<div class="tabs-wr">
						<div class="tab">
							<div class="tab-content">
								<h4 class="title" id="i-reload">
									Sync <span>across all devices</span>
								</h4>
								<p>
									This sounded nonsense to Alice, so she said nothing,
									but set off at once toward the Red Queen. 
									To her surprise, she lost sight of her in a moment.
								</p>
							</div>
						</div>
						<div class="tab">
							<div class="tab-content">
								<h4 class="title" id="i-inbox">
									All emails <span>in one place</span>
								</h4>
								<p>
									This sounded nonsense to Alice, so she said nothing,
									but set off at once toward the Red Queen. 
									To her surprise, she lost sight of her in a moment.
								</p>
							</div>
						</div>
						<div class="tab">
							<div class="tab-content">
								<h4 class="title" id="i-clock">
									Track <span>your time</span>
								</h4>
								<p>
									This sounded nonsense to Alice, so she said nothing,
									but set off at once toward the Red Queen. 
									To her surprise, she lost sight of her in a moment.
								</p>
							</div>
						</div>
					</div>
				</div>
				<div class="column-right">
					<div class="highlights-tablet">
						<img class="img-tablet" src="img/Tablet2.png" alt="tablet-image">
					</div>
				</div>
			</div>
		</div>
	</div>
</section>
<section id="section-getting-started">
	<div class="getting-started">
		<div class="wrapper">
			<div class="columns-wr">
				<div class="column-left">
					<div class="content-wr">
						<h2 class="title">
							Download  <span>to get started</span>
						</h2>
						<p>
							This sounded nonsense to Alice, so she said nothing, 
							but set off at once toward the Red Queen. 
							To her surprise, she lost sight of her in a moment.
						</p>
					</div>
				</div>
				<div class="column-right">
					<div class="buttons-wr">
						<button class="button orange">Buy for $24</button>
						<button class="button white">Try It Free</button>
					</div>
				</div>
			</div>
		</div>
	</div>
</section>
<section id="section-highlights_2">
	<div class="highlights_2">
		<div class="wrapper">
			<div class="highlights_2-content-wr">
				<div class="columns-wr">
					<div class="column-left">
						<div class="col-wr-inner">
							<div class="block block-bg">
								<div class="image-block-wr">
									<img class="block-img" src="img/Tablet_left.png" alt="tablet-image">
								</div>
							</div>
							<div class="block">
								<div class="text-block-wr">
									<h2 class="title">
										Style guide <span>and sample pages included</span>
									</h2>
									<p>
										This sounded nonsense to Alice, so she said <br>
										nothing, but set off at once toward the Red Queen. <br>
										To her surprise, she lost sight of her in a moment, <br>
										and found herself walking.
									</p>
								</div>
							</div>
						</div>
					</div>
					<div class="column-right">
						<div class="col-wr-inner">
							<div class="block">
								<div class="text-block-wr">
									<h2 class="title">
										Perfect <span>landing page for your app</span>
									</h2>
									<p>
										This sounded nonsense to Alice, so she said <br>
										nothing, but set off at once toward the Red Queen. <br>
										To her surprise, she lost sight of her in a moment, <br>
										and found herself walking.
									</p>
								</div>
							</div>
							<div class="block block-bg">
								<div class="image-block-wr">
									<img class="block-img" src="img/Tablet_right.png" alt="tablet-image">
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
	</div>
</section>
<section id="section-video-tour">
	<div class="video-tour">
		<div class="wrapper">
			<div class="video-tour-content-wr">
				<h2 class="title">
					Watch <span>video tour</span>
				</h2>
				<p>
					This sounded nonsense to Alice, so she said nothing, but set off at once toward <br>
					the Red Queen. To her surprise, she lost sight of her.
				</p>
				<button class="play-btn"></button>
			</div>
		</div>
		<div class="video-tour-overlay"></div>
	</div>
</section>
<section id="section-testimonials">
	<div class="testimonials">
		<div class="bg-absolut">
			<div class="bg-relative">
				<div class="bg-left"></div>
				<div class="bg-right"></div>
			</div>
		</div>
		<div class="wrapper">
			<div class="customers-main-wr">
				<div class="customers-left">
					<div class="block-customers">
						<img class="stars" src="icons/stars.png" alt="stars-img">
						<h2 class="title">
							Customers <br> <span>about AppStarter</span>
						</h2>
					</div>
					<div class="block-customers desctop">
						<p>
							Read more reviews:
						</p>
						<ul class="links">
							<li>
								<a href="https://www.apple.com/ru/ios/app-store/">App Store</a>
							</li>
							<li>
								<a href="https://play.google.com">Google Play</a>
							</li>
						</ul>
					</div>
				</div>
				<div class="custumers-right">
					<div class="reviews-wrapper">
						<div class="review">
							<div class="review-inner-wr">
								<div class="review-content">
									<div class="review-avatar-wr">
										<img class="review-avatar" src="icons/user1.png" alt="avatar">
									</div>
									<div class="review-comment">
										<p>
											This sounded nonsense to Alice, so she said nothing, 
											but set off at once toward the Red Queen. 
											To her surprise, she lost sight of her in 
											a moment, and found herself.
										</p>
									</div>
								</div>
								<div class="review-bottom">
									<p class="user-name">
										Eliza Paul<span>- Web Designer </span>
									</p>
									<div class="link-icon">
										<a href="https://twitter.com/?lang=ru"><img src="icons/twitter_icon.png" alt="twitter-icon"></a>
									</div>
								</div>
							</div>
						</div>
						<div class="review">
							<div class="review-inner-wr">
								<div class="review-content">
									<div class="review-avatar-wr">
										<img class="review-avatar" src="icons/user2.png" alt="avatar">
									</div>
									<div class="review-comment">
										<p>
											This sounded nonsense to Alice, so she said nothing, 
											but set off at once toward the Red Queen. 
											To her surprise, she lost sight of her in 
											a moment, and found herself.
										</p>
									</div>
								</div>
								<div class="review-bottom">
									<p class="user-name">
										Juan Richards<span>- Photographer </span>
									</p>
									<div class="link-icon">
										<a href="https://twitter.com/?lang=ru"><img src="icons/twitter_icon.png" alt="twitter-icon"></a>
									</div>
								</div>
							</div>
						</div>
						<div class="review">
							<div class="review-inner-wr">
								<div class="review-content">
									<div class="review-avatar-wr">
										<img class="review-avatar" src="icons/user3.png" alt="avatar">
									</div>
									<div class="review-comment">
										<p>
											This sounded nonsense to Alice, so she said nothing, 
											but set off at once toward the Red Queen. 
											To her surprise, she lost sight of her in 
											a moment, and found herself.
										</p>
									</div>
								</div>
								<div class="review-bottom">
									<p class="user-name">
										Marian McGuire<span>- Entrepreneur</span>
									</p>
									<div class="link-icon">
										<a href="https://twitter.com/?lang=ru"><img src="icons/twitter_icon.png" alt="twitter-icon"></a>
									</div>
								</div>
							</div>
						</div>
					</div>
					<div class="block-customers mobile">
						<p>
							Read more reviews:
						</p>
						<ul class="links">
							<li>
								<a href="https://www.apple.com/ru/ios/app-store/">App Store</a>
							</li>
							<li>
								<a href="https://play.google.com">Google Play</a>
							</li>
						</ul>
					</div>
				</div>
			</div>
		</div>
	</div>
</section>
<section id="section-pricing-table">
	<div class="pricing-table">
		<div class="wrapper">
			<div class="price-col-wr">
				<div class="price-left-col">
					<div class="price-left-content">
						<h2 class="title">
							Choose <span>your plan</span>
						</h2>
						<p>
							This sounded nonsense to Alice, so she said <br>
							nothing, but set off at once toward the Red Queen. <br>
							To her surprise, she lost sight of her in a moment, <br>
							and found herself walking.
						</p>
					</div>
				</div>
				<div class="price-right-col">
					<div class="plans-wr">
						<div class="card proff-card">
							<div class="plan-line proff">
								<h5>Professional</h5>
							</div>
							<h4 class="plan-title proff">10$</h4>
							<ul class="plan-list proff">
								<li>Hundreds of elements</li>
								<li>Style guide</li>
								<li>160 icons included</li>
								<li>Sketch files</li>
								<li>Commercial licence</li>
							</ul>
							<button class="button orange">Buy Now</button>
						</div>
						<div class="card starter-card">
							<div class="plan-line starter">
								<h5>starter</h5>
							</div>
							<h4 class="plan-title starter">FREE</h4>
							<ul class="plan-list starter">
								<li>Hundreds of elements</li>
								<li>Style guide</li>
								<li>30 icons included</li>
								<li>Personal licence</li>
							</ul>
							<button class="button green">Get Free</button>
						</div>
					</div>
				</div>
			</div>
		</div>
	</div>
</section>
<section id="section-contact">
	<div class="contact">
		<div class="wrapper">
			<div class="contact-inner-wr">
				<div class="contact-text">
					<h2 class="title">
						Stay <span>in touch</span>
					</h2>
					<p>
						This sounded nonsense to Alice, so she said nothing, but set off at once toward <br>
						the Red Queen. To her surprise, she lost sight of her in a moment.
					</p>
				</div>
				<div class="form-wr">
					<form>
						<div class="form-inner">
							<div class="inputs-wr">
								<input type="text" id="name" placeholder="Your Name">
								<input type="text" id="e-mail" placeholder="Email Address">
							</div>
							<textarea name="message" id="message" placeholder="Message..." maxlength="1000"></textarea>
							<button class="button orange">Send Message</button>
						</div>
					</form>
				</div>
			</div>
		</div>
	</div>
</section>
<section id="section-footer">
	<div class="footer">
		<div class="wrapper">
			<div class="footer-inner">
				<h5>AppStarter</h5>
				<div class="creator-wr">
					<div class="creator-block">
						<div class="creator-inner">
							<p>Created with</p>
							<img class="heart-icon" src="icons/heart_icon.png" alt="heart_icon">
							<p>by</p> 
							<h6>Sergey Azovskiy</h6>
						</div>
					</div>
					<div class="creator-block">
						<p>© AppStarter, 2017</p>
					</div>
				</div>
				<div class="social-footer">
					<div class="foot-icon" id="twitter">
						<a href="https://twitter.com/?lang=ru" target="_blank"><img src="icons/twitter_icon.png" alt="twitter_icon"></a>
					</div>
					<div class="foot-icon" id="facebook">
						<a href="https://m.facebook.com/" target="_blank"><img src="icons/facebook_Icon.png" alt="facebook_icon"></a>
					</div>
					<div class="foot-icon" id="insta">
						<a href="https://www.instagram.com/?hl=ru" target="_blank"><img src="icons/Instagram_Icon.png" alt="Instagram_icon"></a>
					</div>
				</div>
			</div>
		</div>
	</div>
</section>

<script src="js/jquery-3.3.1.min.js"></script>
<script src="js/main.js"></script>	
</body>
</html>