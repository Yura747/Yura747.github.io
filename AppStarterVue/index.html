<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon href=/AppStarterVue/favicon.ico><title>app-starter-vue</title><link href=/AppStarterVue/css/app.898e656a.css rel=preload as=style><link href=/AppStarterVue/css/chunk-vendors.25d7cad0.css rel=preload as=style><link href=/AppStarterVue/js/app.32c88d80.js rel=preload as=script><link href=/AppStarterVue/js/chunk-vendors.3a57982f.js rel=preload as=script><link href=/AppStarterVue/css/chunk-vendors.25d7cad0.css rel=stylesheet><link href=/AppStarterVue/css/app.898e656a.css rel=stylesheet></head><body><noscript><strong>We're sorry but app-starter-vue doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/AppStarterVue/js/chunk-vendors.3a57982f.js></script><script src=/AppStarterVue/js/app.32c88d80.js></script></body></html>