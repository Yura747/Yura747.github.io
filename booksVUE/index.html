<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon href=/booksVUE/favicon.ico><title>books</title><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900"><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css><link href=/booksVUE/css/chunk-10813b27.ffafaade.css rel=prefetch><link href=/booksVUE/css/chunk-16f95563.cc2729d7.css rel=prefetch><link href=/booksVUE/css/chunk-292705e0.d4eeb05d.css rel=prefetch><link href=/booksVUE/css/chunk-9d2e6c8a.2101e7d4.css rel=prefetch><link href=/booksVUE/js/chunk-10813b27.269e3127.js rel=prefetch><link href=/booksVUE/js/chunk-16f95563.8380a174.js rel=prefetch><link href=/booksVUE/js/chunk-292705e0.62c4c4d6.js rel=prefetch><link href=/booksVUE/js/chunk-9d2e6c8a.401402eb.js rel=prefetch><link href=/booksVUE/css/app.6a927378.css rel=preload as=style><link href=/booksVUE/css/chunk-vendors.3f18537f.css rel=preload as=style><link href=/booksVUE/js/app.b6757802.js rel=preload as=script><link href=/booksVUE/js/chunk-vendors.644cceea.js rel=preload as=script><link href=/booksVUE/css/chunk-vendors.3f18537f.css rel=stylesheet><link href=/booksVUE/css/app.6a927378.css rel=stylesheet></head><body><noscript><strong>We're sorry but books doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/booksVUE/js/chunk-vendors.644cceea.js></script><script src=/booksVUE/js/app.b6757802.js></script></body></html>