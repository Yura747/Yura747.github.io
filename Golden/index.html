<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	
	<title>Golden</title>

	<!-- favicon -->
	<link rel="apple-touch-icon" sizes="76x76" href="icons/logo/apple-touch-icon.png">
	<link rel="icon" type="image/png" sizes="32x32" href="icons/logo/favicon-32x32.png">
	<link rel="icon" type="image/png" sizes="16x16" href="icons/logo/favicon-16x16.png">
	<link rel="manifest" href="icons/logo/site.webmanifest">
	<link rel="mask-icon" href="icons/logo/safari-pinned-tab.svg" color="#5bbad5">
	<meta name="msapplication-TileColor" content="#ffc40d">
	<meta name="theme-color" content="#ffffff">

	<link rel="stylesheet" href="css/jquery-ui.min.css">
	<link rel="stylesheet" href="css/aos.css">
	<link rel="stylesheet" href="css/style-aos.css">
	
	<link rel="stylesheet" href="css/style.css">
</head>
<body>

<div class="header">
	<a href="#" class="logo"><img src="icons/logo.png" alt="No image"></a>
	<ul class="menu" id='menu' data-offset-top="120" data-spy="affix">
		<li class="desktop"><a href="#home">home</a></li>
		<li class="desktop"><a href="#services">services</a></li>
		<li class="desktop"><a href="#portfolio">portfolio</a></li>
		<li class="desktop"><a href="#about">about</a></li>
		<li class="desktop"><a href="#contact">contact</a></li>
	</ul>

	<div class="hamburger-wrapper">
	</div>
</div>
<div class="mobile-menu-wrapper">
	<div class="mobile-menu">
		<ul>
			<li><a href="#home">home</a></li>
			<li><a href="#services">services</a></li>
			<li><a href="#portfolio">portfolio</a></li>
			<li><a href="#about">about</a></li>
			<li><a href="#contact">contact</a></li>
		</ul>
	</div>
</div>
<div class="main" id="home">
	<div class="bg"></div>
	<div class="welcome-text">
		<h2>Welcome To Our Studio!</h2>
		<h1>It's nice to meet you</h1>
		<div class="button">
			<a href="#">tell me more</a>
		</div>
	</div>
</div>
<div class="content">
	<div class="services" id="services">
			<h2>services</h2>
			<h4>Proin iaculis purus consequat sem cure.</h4>
			<div class="wrapper">
				<div class="grid">
					<div class="col">
						<div class="card-services">
							<img src="img/E-Commerce.png" alt="No image">
							<h3>E-Commerce</h3>
							<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt.
							</p>
						</div>
					</div>
					<div class="col">
						<div class="card-services">
							<img src="img/Responsive Web.png" alt="No image">
							<h3>Responsive Web</h3>
							<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt.
							</p>
						</div>
					</div>
					<div class="col">
						<div class="card-services">
							<img src="img/Web Security.png" alt="No image">
							<h3>Web Security</h3>
							<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt.
							</p>
						</div>
					</div>
				</div>
			</div>
	</div>
	<div class="portfolio" id="portfolio">
		<h2>Our	porfolio</h2>
		<h4>Proin iaculis purus consequat sem cure.</h4>
		<div class="wrapper">
			<div class="grid">
				<div class="col">
					<div class="work-card">
						<div class="work-image">
							<img src="img/Image1.png" alt="No Image">
						</div>
						<div class="overlay">
							<div class="overlay-content">
								<h3>Some title</h3>
								<p>
									Lorem ipsum dolor 
									sit amet consectetur 
									adipisicing elit. 
									Iure earum tempore 
									ut enim animi
								</p>
							</div>
						</div>
						<h3>Ebony & Ivory</h3>
						<p>Print Design</p>
					</div>
				</div>
				<div class="col">
					<div class="work-card">
						<div class="work-image">
							<img src="img/Image2.png" alt="No Image">
						</div>
						<div class="overlay">
							<div class="overlay-content">
								<h3>Some title</h3>
								<p>
									Lorem ipsum dolor 
									sit amet consectetur 
									adipisicing elit. 
									Iure earum tempore 
									ut enim animi
								</p>
							</div>
						</div>
						<h3>Smart Stationary</h3>
						<p>Branding</p>
					</div>
				</div>
				<div class="col">
					<div class="work-card">
						<div class="work-image">
							<img src="img/Image3.png" alt="No Image">
						</div>
						<div class="overlay">
							<div class="overlay-content">
								<h3>Some title</h3>
								<p>
									Lorem ipsum dolor 
									sit amet consectetur 
									adipisicing elit. 
									Iure earum tempore 
									ut enim animi
								</p>
							</div>
						</div>
						<h3>Clever Poster </h3>
						<p>Print Design</p>
					</div>
				</div>
				<div class="col">
					<div class="work-card">
						<div class="work-image">
							<img src="img/Image4.png" alt="No Image">
						</div>
						<div class="overlay">
							<div class="overlay-content">
								<h3>Some title</h3>
								<p>
									Lorem ipsum dolor 
									sit amet consectetur 
									adipisicing elit. 
									Iure earum tempore 
									ut enim animi
								</p>
							</div>
						</div>
						<h3>Vinyl Record</h3>
						<p>Product Mock-Up</p>
					</div>
				</div>
				<div class="col">
					<div class="work-card">
						<div class="work-image">
							<img src="img/Image5.png" alt="No Image">
						</div>
						<div class="overlay">
							<div class="overlay-content">
								<h3>Some title</h3>
								<p>
									Lorem ipsum dolor 
									sit amet consectetur 
									adipisicing elit. 
									Iure earum tempore 
									ut enim animi
								</p>
							</div>
						</div>
						<h3>Treehouse Template</h3>
						<p>Web Design</p>
					</div>
				</div>
				<div class="col">
					<div class="work-card">
						<div class="work-image">
							<img src="img/Image6.png" alt="No Image">
						</div>
						<div class="overlay">
							<div class="overlay-content">
								<h3>Some title</h3>
								<p>
									Lorem ipsum dolor 
									sit amet consectetur 
									adipisicing elit. 
									Iure earum tempore 
									ut enim animi
								</p>
							</div>
						</div>
						<h3>Burned Logo</h3>
						<p>Branding</p>
					</div>
				</div>
			</div>
		</div>
	</div>	
	<div class="about" id="about">
		<h2>About us</h2>
		<h4>Proin iaculis purus consequat sem cure.</h4>
		<div class="info-block-wr">
			<div class="info-block">
				<div class="block-photo">
					<a href="#"><img src="img/Image7.png" alt="No image"></a>
				</div>
				<div class="block-content-wr">
					<div class="block-content">
						<h3>July 2010</h3>
						<h4>Our Humble Beginningsr</h4>
						<p>Proin iaculis purus consequat sem cure <br>
							digni ssim. Donec porttitora entum suscipit <br> 
							aenean rhoncus posuere odio in tincidunt. Proin <br>
							iaculis purus consequat sem cure digni <br>
							ssim. Donec porttitora entum suscipit.
						</p>
					</div>
				</div>
			</div>
			<div class="info-block">
				<div class="block-photo">
					<a href="#"><img src="img/Image8.png" alt="No image"></a>
				</div>
				<div class="block-content-wr">
					<div class="block-content">
						<h3>January 2011</h3>
						<h4>Facing Startup Battles</h4>	
						<p>Proin iaculis purus consequat sem cure <br>
							digni ssim. Donec porttitora entum suscipit <br>
							aenean rhoncus posuere odio in tincidunt. Proin <br>
							iaculis purus consequat sem cure digni <br>
							ssim. Donec porttitora entum suscipit aenean <br>
							rhoncus posuere odio in tincidunt.
						</p>
					</div>
				</div>
			</div>
			<div class="info-block">
				<div class="block-photo">
					<a href=""><img src="img/Image9.png" alt="No image"></a>
				</div>
				<div class="block-content-wr">
					<div class="block-content">
						<h3>December 2012</h3>
						<h4>Enter The Dark Days</h4>							
						<p>Proin iaculis purus consequat sem cure <br>	
							digni ssim. 
							Donec porttitora entum suscipit <br>
							aenean rhoncus posuere odio in tincidunt. 
							Proin <br>
							iaculis purus consequat sem cure digni.
						</p>
					</div>
				</div>
			</div>
			<div class="info-block">
				<div class="block-photo">
					<a href="#"><img src="img/Image10.png" alt="No image"></a>
				</div>
				<div class="block-content-wr">
					<div class="block-content">
						<h3>february 2014</h3>
						<h4>Our Triumph</h4>								
						<p>Proin iaculis purus consequat sem cure <br>
							digni ssim. Donec porttitora entum suscipit <br>
							aenean rhoncus posuere odio in tincidunt. Proin <br>
							iaculis purus consequat sem cure digni <br>
							ssim. Donec porttitora entum suscipit aenean.
						</p>
					</div>
				</div>
			</div>
			<div class="info-block" id="block-btn">
				<div class="block-photo">
					<a href="#">Our <br>
								Story <br>
								Continues <br>
								 ...</a>
				</div>
				<div class="block-content-wr">
					<div class="block-content">
						<h3>							
						</h3>
						<p>
						</p>
					</div>
				</div>
			</div>
			<div class="info-block">
				<div class="block-photo">
					<a href="#"><img src="img/Image4.png" alt="No image"></a>
				</div>
				<div class="block-content-wr">
					<div class="block-content">
						<h3>Some new block</h3>
						<h4>Our Triumph</h4>								
						<p>Lorem ipsum dolor sit amet consectetur <br> 
						   adipisicing elit. Reprehenderit illum, <br>
						   minus quas soluta dolor veritatis <br>
						   nam nihil sapiente ea debitis. <br>
						   Laboriosam eveniet minus similique  <br>	
						   blanditiis iure fugiat fuga quasi iste.
						</p>
					</div>
				</div>
			</div>
			<div class="info-block">
				<div class="block-photo">
					<a href="#"><img src="img/Image6.png" alt="No image"></a>
				</div>
				<div class="block-content-wr">
					<div class="block-content">
						<h3>Some new block</h3>
						<h4>Our Triumph</h4>								
						<p>Lorem ipsum dolor sit amet <br>
							consectetur adipisicing elit.<br>  
							Reprehenderit illum, minus quas soluta <br>
							dolor veritatis nam nihil sapiente ea <br>
							debitis. Laboriosam eveniet minus <br>
							similique blanditiis iure fugiat <br>
							fuga quasi iste.
						</p>
					</div>
				</div>
			</div>
		</div>
	</div>
	<div class="team">
		<h2>Our amazing team</h2>
		<h4>Proin iaculis purus consequat sem cure.</h4>
		<div class="wrapper">
				<div class="grid">
					<div class="col">
						<div class="team-member-card">
							<!-- <div class="photo">
							</div> -->
							<div class="wrapp-rotate">  <!--t3d-->
								<div class="photo-rotate"> <!-- rotate-->
										<img src="img/Image13.png" alt="No image">
										<div class="text-rotate-wr">  <!-- text -->
											<div class="text-rotate"> <!-- pd  -->
												<p>Lorem, ipsum dolor 
												   sit amet consectetur 
												   adipisicing elit. 
												</p>
											</div>
										</div>
								</div>
							</div>
							<h3>Kimberly Thompson</h3>
							<p>Marketer</p>
							<div class="social-wrapper">
								<div class="social-img" id="twitter"></div>
								<div class="social-img" id="facebook"></div>
								<div class="social-img" id="pinterest"></div>
							</div>
						</div>
					</div>
					<div class="col">
						<div class="team-member-card">
							<!-- <div class="photo">
							</div> -->
							<div class="wrapp-rotate">  <!--t3d-->
								<div class="photo-rotate"> <!-- rotate-->
										<img src="img/Image12.png" alt="No image">
										<div class="text-rotate-wr">  <!-- text -->
											<div class="text-rotate"> <!-- pd  -->
												<p>Lorem, ipsum dolor 
												   sit amet consectetur 
												   adipisicing elit. 
												</p>
											</div>
										</div>
								</div>
							</div>
							<h3>Ricco Massimo</h3>
							<p>Coder</p>
							<div class="social-wrapper">
								<div class="social-img" id="twitter"></div>
								<div class="social-img" id="facebook"></div>
								<div class="social-img" id="google"></div>
							</div>
						</div>
					</div>
					<div class="col">
						<div class="team-member-card">
							<!-- <div class="photo">
							</div> -->
							<div class="wrapp-rotate">  <!--t3d-->
								<div class="photo-rotate"> <!-- rotate-->
										<img src="img/Image11.png" alt="No image">
										<div class="text-rotate-wr">  <!-- text -->
											<div class="text-rotate"> <!-- pd  -->
												<p>Lorem, ipsum dolor 
												   sit amet consectetur 
												   adipisicing elit. 
												</p>
											</div>
										</div>
								</div>
							</div>
							<h3>Uku Mason</h3>
							<p>Graphic Designer</p>
							<div class="social-wrapper">
								<div class="social-img" id="facebook"></div>
								<div class="social-img" id="pinterest"></div>
							</div>
						</div>
					</div>
					<p>Proin iaculis purus consequat sem cure  digni 
							ssim donec porttitora entum suscipit  
							aenean rhoncus posuere odio in 
							tincidunt proin iaculis.
					</p>
				</div>
		</div>
	</div> 
</div>
<div class="sponsors">
	<div class="sponsor-image">
		<img src="icons/Envato1.png" alt="No image">
	</div>
	<div class="sponsor-image">
		<img src="icons/WordPress.png" alt="No image">
	</div>
	<div class="sponsor-image">
		<img src="icons/Tuts+.png" alt="No image">
	</div>
	<div class="sponsor-image">
		<img src="icons/Microlancer copy.png" alt="No image">
	</div>
</div>

<div class="contact" id="contact">
	<h2>Contact us</h2>
	<h4>Proin iaculis purus consequat sem cure.</h4>
	<div class="form-wrapper">
		<div class="form">
			<input type="text" placeholder="your name *">
			<input type="email" placeholder="your e-mail *">
			<input type="text" placeholder="subject">
		</div>
		<div class="form">
			<textarea id="textarea_id" maxlength="216" placeholder="your message *"></textarea>
		</div>
	</div>
	<div class="button">
		<a href="#">Send message</a>
	</div>
</div>
<div class="footer">
		<div class="social-wrapper">
			<div class="social-img" id="twitter"></div>
			<div class="social-img" id="facebook"></div>
			<div class="social-img" id="pinterest"></div>
			<div class="social-img" id="google"></div>
		</div>
</div>


<div class="swipe" data-aos="fade-left" data-aos-anchor="#example-anchor" data-aos-offset="2500" data-aos-duration="1000">
	<img src="icons/swipe.png" alt="No image">
</div>

<script src="js/jquery-3.3.1.min.js"></script>
<script src="js/golden-mobile-menu.js"></script>	 
<script src="js/jquery.js"></script>
<script src="js/jquery-ui.min.js"></script>
<script src="js/aos.js"></script>
<script src="js/main-aos.js"></script>

<script src="js/main.js"></script>	
</body>
</html>